client/notices.go

```diff
@@ -33,6 +34,11 @@ type NotifyOptions struct {
 	// "domain.com/key" format.
 	Key string
 
+	// UserID is the UID of the notice's intended recipient. A UID of -1 means
+	// the notice may be received by any user. If nil, uses the default notice
+	// recipient UID.
```

Comment:
I think we need to say what the default is here (and change it in Snapd if it's going to be different there):

```suggestion
	// UserID is the UID of the notice's intended recipient. A UID of -1 means
	// the notice may be received by any user (also the default if UserID is nil).
```